### PR TITLE
Saner default message

### PR DESCRIPTION
**Pull request by karottenreibe**
Addresses issue https://jira.cqse.eu/browse/TS-24326

- [x] Changes are tested adequately
- [x] Agent's README.md updated in case of user-visible changes
- [x] CHANGELOG.md updated
- [x] Present new features in [N&N](https://wiki.cqse.eu/pages/viewpage.action?pageId=689566)
- [x] [TGA Tutorial](https://docs.teamscale.com/tutorial/setting-up-tga-java) updated
- [x] [TIA Tutorial](https://docs.teamscale.com/tutorial/tia-java/) updated

Please respect the vote of the [Teamscale bot](https://demo.teamscale.com) or flag irrelevant findings as tolerated or false positives. If you feel that the Teamscale config needs adjustment, please state so in a comment and discuss this with your reviewer.

### CODE REVIEW

File: teamscale-gradle-plugin/src/test/kotlin/com/teamscale/TeamscalePluginTest.kt [hunk 1]
```diff
@@ -135,7 +135,7 @@ class TeamscalePluginTest {
             .withProjectDir(temporaryFolder.root)
             .withPluginClasspath()
             .withArguments(runnerArgs)
-            .withGradleVersion("4.6")
+            .withGradleVersion("6.5")
```
Comment (DreierF) [inline teamscale-gradle-plugin/src/test/kotlin/com/teamscale/TeamscalePluginTest.kt:138]: Why did you need to update the gradle version here?

4.6 is our minimal supported version and we run the tests against this version to ensure everything still works on this version.
When updating we should also adjust https://github.com/cqse/teamscale-jacoco-agent/blob/master/teamscale-gradle-plugin/src/main/kotlin/com/teamscale/TeamscalePlugin.kt#L50

**Review by DreierF — CHANGES_REQUESTED** [review]

File: teamscale-gradle-plugin/src/test/kotlin/com/teamscale/TeamscalePluginTest.kt (hunk 1 above)
Comment (karottenreibe) [reply teamscale-gradle-plugin/src/test/kotlin/com/teamscale/TeamscalePluginTest.kt:138]: Ah I see. I was unable to run the tests on my machine since Gradle 4.6 seems to not run under Java 11, so I just updated the version to match what we use elsewhere. What's your opinion? Leave at 4.6 or bump to the first Gradle version that supports Java 11?

Comment (DreierF) [reply teamscale-gradle-plugin/src/test/kotlin/com/teamscale/TeamscalePluginTest.kt:138]: Bump to Java 11 is fine for me. We are soon on 6.6.1 so 6.5 would be fine.

**Comment by karottenreibe** [comment]
I added a commit that also includes the hostname in the default message

File: teamscale-gradle-plugin/src/test/kotlin/com/teamscale/TeamscalePluginTest.kt (hunk 1 above)
Comment (DreierF) [reply teamscale-gradle-plugin/src/test/kotlin/com/teamscale/TeamscalePluginTest.kt:138]: Could you please update the version in TeamscalePlugin (see link above)?

**Review by DreierF — APPROVED** [review]
